packages/shared/setupTests.ts

```diff
@@ -1,8 +1,11 @@
 import 'regenerator-runtime/runtime'
-
 import { pointerEventPolyfill } from './pointerEventPolyfill'
 
+// Let React know that we'll be testing effectful components
 // @ts-ignore
 global.IS_REACT_ACT_ENVIRONMENT = true
```

Comment:
We should probably document the usage of `IS_REACT_ACT_ENVIRONMENT` for React 18 since that was very hard to find.

Can probably include that in the testing docs as well for a setup guide.